docs/source/notebooks/iv_pymc.ipynb

```diff
@@ -0,0 +1,4160 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    # %config InlineBackend.figure_format = 'svg'
</pre><p>shall we user "retina" ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Probably a good idea. I was experimenting with sag at one point to try to keep file sizes down. But I have a vague memory that svg had some rendering issues on either GitHub or readthedocs.